website/docs/api/Mock.md

```diff
@@ -111,3 +111,55 @@ interface MatchEvent {
 ### `continue`
 
 This event is being emitted when the network response has neither been overwritten nor interrupted, or if response was already sent by another mock. `requestId` is passed in event callback.
+
+### Examples
+
+Getting a number of pending requests:
+
+```js
+let pendingRequests = 0
+const mock = await browser.mock('**')
```

Comment:
Let's add a comment here to emphasise it is important to match all requests otherwise, the resulting value can be very confusing.